### PR TITLE
[RM ch02] profiles figure update

### PR DESCRIPTION
**Pull request by xavier-grall**
Replace PR #531 to also consider issue #536 (in addition to issue #344 and PR suggested by @ASawwaf in #350)
@ulikleber @kedmison @ASawwaf @pgoyal01 
Thank you to approve

### CODE REVIEW

File: doc/ref_model/chapters/chapter02.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Based on the above analysis, following NFVI profiles are proposed (Also shown in
 
 - **Basic**: VNFCs that perform basic compute operations.
 - **Network Intensive**: VNFCs that perform network and compute intensive operations with high throughput and low latency requirements.
-- **Compute Intensive**: VNFCs that perform compute intensive operations.
+- **Compute Intensive**: VNFCs that perform compute intensive operations (but do not have high network throughput and low latency requirements).
 - **Storage Intensive**: VNFCs that perform storage intensive operations with high IOPS requirements. (_**Note:** Storage Intensive Profile will not be defined in initial CNTT releases_)
```
Comment (ASawwaf) [inline doc/ref_model/chapters/chapter02.md]: @xavier-grall did you mean " but do not have high network throughput and *have* low latency requirements1"

Comment (xavier-grall) [reply doc/ref_model/chapters/chapter02.md]: @ASawwaf  @pgoyal01
No, so the wording needs to be more precise: maybe  "but do not have **neither** high network throughput **nor** low **network** latency requirements"
But it is a proposal from Pankaj, so his opinion is needed.
In my view the low latency required for control functions targeted by that profile is provided at computation level by cpu pinning. But not so sure, and a minimal network acceleration may also be needed (eg, a software acceleration with very few dpdk pmd threads).

Comment (ASawwaf) [reply doc/ref_model/chapters/chapter02.md]: @xavier-grall @pgoyal01  ," but do not have neither high network throughput nor low network latency requirements" this definition is valid for basic or general-purpose profiles 

But for Control plan VNF ( as IMS ) : I think it is required low latency which should be onboarded on Compute intensive profile 

what do u think ?

Comment (xavier-grall) [reply doc/ref_model/chapters/chapter02.md]: In my opinion, low latency requirements of some control plane functions (as IMS CSCF, MME...) are addressed by providing these functions with sufficient, ie dedicated, NFVI computation resources, but not with specific NFVI networking resources. 
And these dedicated computation resources are reflected by the "fast computing" requirement of CI profile (eg, provided by cpu pinning)
But, as I also said previously, maybe I'm wrong and we should also consider some specific networking resources and an additional low latency requirement (eg provided by dpdk-based vswitching).
@pgoyal01, @kedmison, @ulikleber 
Thank you in advance for your opinion on that point.

**Comment by markshostak** [comment]
@rabi-abdel Hi Rabi, This PR is approved, but merging is blocked. Are you critical path to the merge or is there something else required? Thx, -M

**Comment by xavier-grall** [comment]
@markshostak @rabi-abdel 
i guess the merging is blocked because I have requested reviewers to review again for having their opinion on Ahmed proposal (see https://github.com/cntt-n/CNTT/pull/555#discussion_r342925372)

File: doc/ref_model/chapters/chapter02.md (hunk 1 above)
Comment (ulikleber) [reply doc/ref_model/chapters/chapter02.md]: We are talking about generic profiles here. In reality many VNFs (better VNFCs) only roughly fall into one of 3 categories and have specific needs to reach their optimum effectivity.

Comment (kedmison) [reply doc/ref_model/chapters/chapter02.md]: I would suggest wording: "Basic: VNFCs that perform basic compute operations and can tolerate oversubscription the variable compute latency."

**Comment by markshostak** [comment]
Per RM review mtg, Kelvin and Xavier to update language for profiles.

**Comment by markshostak** [comment]
Mehmet to investigate ML/AI and determine if additional profile is required.

File: doc/ref_model/chapters/chapter02.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter02.md]: @ASawwaf I agree with @xavier-grall the control plane latency requirements will be addressed by CPU and memory.

Comment (xavier-grall) [reply doc/ref_model/chapters/chapter02.md]: I suggest to deal with the evolution of profiles requirements and definitions from another issue to create.
And right now, to merge the current PR for taken into account:
- the initial issue by Rabi (CI/NI figures inverting) 
- the example vnf change by Ahmed (PGW on NI)
- the example vnf change by Kelvin (no PCRF on Basic).

**Comment by xavier-grall** [comment]
I suggest to deal with the evolution of profiles requirements and definitions from another issue to create, so that we can merge the PR right now and thus address :
    the initial issue by Rabi (CI/NI figures inverting)
    the example vnf change by Ahmed (PGW on NI)
    the example vnf change by Kelvin (no PCRF on Basic).

**Comment by xavier-grall** [comment]
@kedmison @pgoyal01 
I have just created a new issue #571 for CI & basic profiles language update, so thank you to approve that PR, so that we can merge it (and thus address the initial request from Rabi for inverting CI & NI figures).

**Comment by xavier-grall** [comment]
@markshostak @rabiabdel 
I guess that PR can be merged now.
Since it also includes an update of CNTT_diagrams.pptx in artefacts directory, the merge should be done asap to avoid a conflict with a possible parallel update (but I don't know if there is another PR which could also modify it).

**Pull request merged**